merchantapi/src/test/java/fi/hel/verkkokauppa/configuration/api/merchant/MerchantControllerUnitTest.java

```diff
@@ -63,6 +66,7 @@
         ActiveMQAutoConfiguration.class,
         KafkaAutoConfiguration.class
 })
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
This ensures that the method with **BeforeAll** -annotation is run only once per class lifecycle and not before every test. I decided to mock configuration mapper manually, because AbstractModelMapper makes it a little bit bigger hassle to instantiate it through spring DI (dependency injection).